tests/test_dataManager.py

```diff
@@ -95,7 +95,12 @@ def test_make_primary_hdu(self):
     def test_make_wavelength_hdu(self):
         manager = DataManager(instrument=self.instrument, origin=self.origin)
         hdu = manager.make_wavelength_hdu(self.data)
-        np.testing.assert_array_equal(Table.read(hdu)['wavelength'], self.wavelength)
+        # Need first wavelength from first row
+        wavelengths = QTable.read(hdu)['wavelength'][0]
+        # Should be 2D
+        self.assertEqual(wavelengths.shape, (self.wavelength.size, 1))
+        wavelengths = wavelengths.reshape(wavelengths.size)
+        np.testing.assert_array_equal(wavelengths, self.wavelength)
```

Comment:
These tests don't do anything to confirm that what was written is actually a FITS `-TAB` style WCS. Do we have any way to test that?